README.md

```diff
@@ -123,13 +123,26 @@ window.setTimeout(function () {
 }, splashDuration - fadeDuration);
 ```
 
-### iOS Quirks
+### iOS and Windows Quirks
 
 - `ShowSplashScreenSpinner` (boolean, defaults to `true`): Set to `false`
   to hide the splash-screen spinner.
 
         <preference name="ShowSplashScreenSpinner" value="false"/>
 
+### Windows Quirks
+
+- `SplashScreenSpinnerColor` (string, defaults to system accent color): hash, rgb notation or CSS color name.
+
+        <preference name="SplashScreenSpinnerColor" value="#242424"/>
+        <preference name="SplashScreenSpinnerColor" value="DarkRed"/>
+        <preference name="SplashScreenSpinnerColor" value="rgb(50,128,128)"/>
+
+- `SplashScreenBackgroundColor` (string, defaults to #464646): hex notation.
+
+        <preference name="SplashScreenBackgroundColor" value="0xFFFFFFFF"/>
+
+
```

Comment:
@SunboX, thanks, there is a separate issue for that: https://issues.apache.org/jira/browse/CB-9287